src/docs/layouts/NavigationSPA.jsx

```diff
@@ -0,0 +1,57 @@
+// @flow
```

Comment:
mostly copy-paste of original navigation, but with react-router instead of normal links